iron-doc-viewer-styles.html

```diff
@@ -144,7 +146,7 @@
         margin-bottom: 20px;
       }
 
-      .card > header {
+      .card > header, .card a {
```

Comment:
nit: users may want to use the mixin `--iron-doc-viewer-header` and now it would also apply to the anchors inside the header.  Maybe we can separate them? Finally, I know this CSS was there, but it might be good to remove `padding: 14px 0;` and adjust the `line-height` in `.card > header` instead. wdyt?